apps/astarte_appengine_api/.dockerignore

```diff
@@ -11,6 +11,8 @@ erl_crash.dump
 *.beam
 config/*.secret.exs
 .elixir_ls/
+README.md
+.formatter.exs
```

Comment:
Leftover?

Reply:
Mumble, isn't it better to create a separate section for each project?